src/installation/index.md

```diff
@@ -1,160 +1,204 @@
+
+[`espup`][espup-github] is a tool that simplifies installing and maintaining the components required to develop Rust applications for the `Xtensa` architecture.
 
-`espup` will create an export file, by default `$HOME/export-esp.sh` on Unix systems and `%USERPROFILE%\export-esp.ps1` on Windows. This file contains some environment variables required to build projects. Make sure to source this file in every terminal before building any application:
+[espup-github]: https://github.com/esp-rs/espup
 
+To install `espup`, run:
 ```sh
-# Unix
-. $HOME/export-esp.sh
-# Windows
-%USERPROFILE%\export-esp.ps1
+cargo install espup
 ```
 
-[esp-rs/espup]: https://github.com/esp-rs/espup
-[see Usage section of the `espup` Readme]: https://github.com/esp-rs/espup#usage
+You can also directly download pre-compiled [release binaries] or use [cargo-binstall].
+
 [release binaries]: https://github.com/esp-rs/espup/releases
 [cargo-binstall]: https://github.com/cargo-bins/cargo-binstall
 
+Once `espup` is installed, do the following:
+
+1. Install all the necessary tools to develop Rust applications for all supported Espressif targets by running:
+    ```sh
+    espup install
+    ```
+
+    `espup` will create an export file that contains some environment variables required to build projects:
+
+    - Unix systems - `$HOME/export-esp.sh`
+    - Windows - `%USERPROFILE%\export-esp.ps1`
+
+2. On Unix systems, make sure to source this file in every terminal before building any application: `. $HOME/export-esp.sh`
+
+  > On Windows systems, no need to source the file. It is only created to show the modified environment variables.
+
+
+After running `espup install`:
+
+- `no_std` (bare-metal) applications should work out of the box
+- `std` applications require additional software covered in [`std` Development Requirements][rust-esp-book-std-requirements]
+
 ### Other installation methods
 
 - Using [esp-rs/rust-build] installation scripts. This was the recommended way in the past, but now the installation scripts are feature frozen and all new features will only be included in `espup`. See the repository README for instructions.
-- Building the Rust compiler with `Xtensa` support from source. This process is computationally expensive and can take one or more hours to complete depending on your system. It is not recommended unless there is a major reason to go for this approach. See instructions in the [Installing from Source section of the esp-rs/rust repository].
+- Building the Rust compiler with `Xtensa` support from source. This process is computationally expensive and can take one or more hours to complete depending on your system. It is not recommended unless there is a major reason to go for this approach. Here is the repository to build it from source: [esp-rs/rust repository].
+
+[esp-rs/rust-build]: https://github.com/esp-rs/rust-build#download-installer-in-bash
+[esp-rs/rust repository]: https://github.com/esp-rs/rust
+
+### What espup Installs
```

Comment:
I think that  we should move "What espup Installs" chapter before "Other installation methods"

Reply:
Done!